src/pages/demo.js

```diff
@@ -10,7 +10,17 @@ function Demo() {
 	const iframeEl = useRef();
 
 	useEffect(() => {
-		const FRAME_ORIGIN = 'https://stylelint-demo.herokuapp.com';
+		if (!window.crossOriginIsolated) {
+			// Reload to apply headers for COI (cross-origin isolated)
+			//   This control is not necessary when applying COI to all pages. However,
+			//   some external images cannot be loaded on pages using COI.
+			//   Another reason is that SPA cannot control the header for each page.
+			window.location.reload();
+
+			return;
+		}
+
+		const FRAME_ORIGIN = 'https://deploy-preview-352--chimerical-trifle-8d3c21.netlify.app';
```

Comment:
We probably need to change here:

```suggestion
		const FRAME_ORIGIN = 'https://chimerical-trifle-8d3c21.netlify.app';
```